api/src/main/resources/metadata/Concepts_MCH-1.xml

```diff
@@ -1,3 +1,48 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <dataset>
+  <concept_datatype concept_datatype_id="1" name="Numeric" hl7_abbreviation="NM" description="Numeric value, including integer or float (e.g., creatinine, weight)" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4a4488-c2cc-11de-8d13-0010c6dffd0f"/>
```

Comment:
I initially removed the concept metadata definitions but then I realized that they pose no harm. The data exchange module gracefully ignores them if they already exist. I also think that the metadata should remain so as to guide whoever is looking at the file with naked eyes understand the relationship between the concepts and the metadata definitions. What do you think about that?